src/module/item/entity.js

```diff
@@ -170,85 +159,9 @@ export class OseItem extends Item {
     });
   }
 
-  _getRollTag(data) {
-    if (data.roll) {
-      const roll = `${data.roll}${
-        data.rollTarget ? CONFIG.OSE.roll_type[data.rollType] : ""
-      }${data.rollTarget ? data.rollTarget : ""}`;
-      return {
-        label: `${game.i18n.localize("OSE.items.Roll")} ${roll}`,
-      };
-    } else {
-      return;
-    }
-  }
-
-  _getSaveTag(data) {
-    if (data.save) {
-      return {
-        label: CONFIG.OSE.saves_long[data.save],
-        icon: "fa-skull",
-      };
-    } else {
-      return;
-    }
-  }
-
-  getAutoTagList() {
```

Comment:
this function is still used in `show()`
this file, line ~243
swapping with `this.tags()` no longer throws an error when clicking the eye icon to show an item in chat